internal/command/jsonformat/differ/output.go

```diff
@@ -1,7 +1,86 @@
 package differ
 
-import "github.com/hashicorp/terraform/internal/command/jsonformat/change"
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/internal/command/jsonformat/change"
+	"github.com/hashicorp/terraform/internal/plans"
+	"github.com/zclconf/go-cty/cty"
+)
+
+const (
+	jsonNumber = "number"
+	jsonObject = "object"
+	jsonArray  = "array"
+	jsonBool   = "bool"
+	jsonString = "string"
+	jsonNull   = "null"
+)
```

Comment:
This comment applies multiple times through this series of PRs, but I only remembered it here. It'd be nice to have a trivial named type for these string values. We tend to do this elsewhere when we have a fixed enumeration of values, and it's helpful for code navigation and understanding the semantics of function types.

Reply:
I've made a note of this, and I'll put together another PR that gets rid of any dangling string types.